indexer/workers/importer.py

```diff
@@ -92,8 +88,12 @@ def create_index(self, index_name: str) -> None:
                     mappings=self.mappings,
                     settings=self.settings,
                 )
+                logger.info("Index '%s' created successfully." % index_name)
             else:
                 self.client.indices.create(index=index_name)
+                logger.info("Index '%s' created successfully." % index_name)
```

Comment:
```suggestion
            else:
                self.client.indices.create(index=index_name)
            logger.info("Index '%s' created successfully." % index_name)
```
This should log for either `if` or `else` path I believe.